dhis-2/dhis-web-api-test/pom.xml

```diff
@@ -21,6 +21,11 @@
         <artifactId>maven-dependency-plugin</artifactId>
         <configuration>
           <failOnWarning>true</failOnWarning>
+          <ignoredUnusedDeclaredDependencies>
+            <ignoredUnusedDeclaredDependency>com.h2database:h2</ignoredUnusedDeclaredDependency>
+            <ignoredUnusedDeclaredDependency>org.skyscreamer:jsonassert</ignoredUnusedDeclaredDependency>
```

Comment:
Why do we need to ignore it as unused declared? Its imported in TrackerImportParamsSerdeTest 🤔

Reply:
I need to comment on the ignore unused as they are not intuitive at all.
On this particular one, I now removed the runtime dependency.
It was happening in https://github.com/dhis2/dhis2-core/blob/8298dbaa92923ece2228bb5e9516ae4d65d9d4ef/dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/StaticContentControllerTest.java#L209

Changing the json assert to a string assert removed the need to have `jsonassert` dependency.
The error was `java.lang.NoClassDefFoundError: org/skyscreamer/jsonassert/JSONAssert`